src/time_man.cpp

```diff
@@ -258,6 +258,18 @@ String time_format_time(tm &time)
   return String(output);
 }
 
+
+// inspired from https://www.snad.cz/en/2018/12/21/uptime-and-esp8266/
+// this works if this method is called at least once every 47 days, which is the case (each 8 hours at least based on TIME_POLL_TIME)
```

Comment:
There is no direct link to `TIME_POLL_TIME` and calling `TimeManager::serialise` and although chances are high that it would be called, you may want to add something explicit into the `loop` method. Also be aware that the `loop` method is only called regularly if SNTP is enabled.

Reply:
True, I will add a more direct link because here, it works, but I rely on the whole call hierarchy.

@jeremypoulter Regarding the loop which could be disabled...

- should we then move the update stuff in its own cpp file with its own microtask ?
- should we couple the uptime code to another existing loop, which is less time-related ?
- should we keep the uptime code here, and accept the fact that time-related things need NTP ?

Reply:
@jeremypoulter : I've updated the PR to move the uptime near the main loop, where the version and hardware info are.